autoPyTorch/pipeline/components/training/data_loader/base_data_loader.py

```diff
@@ -27,8 +28,12 @@ class BaseDataLoaderComponent(autoPyTorchTrainingComponent):
     https://pytorch.org/docs/stable/data.html
 
     """
+    _fit_requirements = [FitRequirement("dataset", str), FitRequirement("root", str),
```

Comment:
Can we have one requirement per line for better visibility?

Reply:
Sure